README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
I run server first, after server is up and running, then run client.  My point is that server should not related on any function on client side.

Like we discussed before, maybe we can separate client to two parts: operator  and robot/sensor controller.
1. operator runs RVIZ 
2. Robot/sensor controller runs real robot driver or simulated controller/sensor.

When running the server, we can require that Robot/sensor controller needs to run first.
but operator(RVIZ) can run anytime after server and Robot/sensor controller are up and running.